engine/api/src/main/java/pl/touk/nussknacker/engine/api/editor/SimpleEditorType.java

```diff
@@ -2,5 +2,5 @@
 
 public enum SimpleEditorType {
     BOOL_EDITOR, STRING_EDITOR, FIXED_VALUES_EDITOR, DATE_EDITOR, TIME_EDITOR, DATE_TIME_EDITOR,
-    DURATION_EDITOR, PERIOD_EDITOR
+    DURATION_EDITOR, PERIOD_EDITOR, CRON_EDITOR
```

Comment:
why don't create each enum value in new line?

Reply:
Whe create?:)

Reply:
It follows project convention, see JavaSampleEnum, TariffType. But maybe it will be more readable...

Reply:
I've moved each enum value to separate line